dbt/scripts/export_qc_town_close_reports.py

```diff
@@ -181,14 +181,17 @@ def main():
 
             iasworld_deps[table_name] = formatted_dep
 
-        print("Run the following commands on the Data Team server:")
+        print(
+            "Run the following commands on the Data Team",
+            "server as the shiny-server user:",
+        )
         print()
         print("cd /home/shiny-server/services/service-spark-iasworld")
-        print("docker compose up -d")
+        print("docker compose --profile prod up -d")
         print(
-            "docker exec spark-node-master ./submit.sh "
-            "--no-run-github-workflow "
-            f"--json-string '{json.dumps(iasworld_deps)}' "
+            "docker exec spark-node-master-prod ./submit.sh",
```

Comment:
**[Nitpick, non-blocking]** I believe that if someone copy-pastes this output, bash will assume that each line is a new commad, is that right? If so, can we append forward slashes to the end of each line break?

Reply:
Ah, no line breaks here, the whole thing is just a single long line.

Reply:
Ah, my mistake. Thanks!